### PR TITLE
Added docker configs for all essential services + docker compose setup for the full stack

### PR DESCRIPTION
**Pull request by grzkv**
see title + commit messages

Now the full stack of 
- `go-carbon`
- `carbonzipper`
- `carbonapi`
can be run with single `docker-compose`

### CODE REVIEW

**Comment by grzkv** [comment]
Fixes #58

File: README.md [hunk 1]
```diff
@@ -34,9 +34,25 @@ make debug
 We do not provide packages for install at this time. Contact us if you're
 interested in those.
 
-### Run
+## Run
 
-You can easily run the program with sample config provided in `config/` directory by running `./run.sh`.
+Run the full stack `carbonapi` -> `zipper` -> `go-carbon` with:
```
Comment (azhiltsov) [inline README.md]: in this setup you don't need a zipper.
carbonapi -> go-carbon should work fine. Zipper needed if you have more than 1 go-carbon instance.

Comment (gksinghjsr) [reply README.md]: No, we should have our setup as the default, and not the simplest stack missing a key component.

Comment (grzkv) [reply README.md]: It makes sense to have a simplified setup as a special case.

But in general case, we need `zipper` to have a full stack for testing changes and running system tests.

**Pull request merged**